src/main/resources/edu/cornell/kfs/paymentworks/businessobject/datadictionary/PaymentWorksVendor.xml

```diff
@@ -8,10 +8,11 @@
 
 	<bean id="PaymentWorksVendor-parentBean" abstract="true"
```

Comment:
What is the relationship between PaymentWorksVendor and Vendor? PaymentWorksVendor has separate lookup and inquiry screens?